bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/README.md

```diff
@@ -226,12 +295,72 @@ setting the properties for the job. For example:
     -Dhbase.snapshot.thread.pool.max=10
     ```
 
+### Example jobs to validate the data migrated from source to target
+
+1. Set the following additional environment variables for running the validation steps.
+    ```bash
+    # hash-table validation job
+    export HBASE_TABLENAME=<HBASE_TABLENAME>
+    # hbase zookeeper quorum (ie. zk1.example.com:2181)
+    export HBASE_ZK_QUORUM=<ZK_QUORUM>
+    export HASH_OUTPUTDIR=<HASH_OUTPUTDIR>
+
+    # sync-table validation job
+    export HBASE_ZK_QUORUM_FULL=${HBASE_ZK_QUORUM}:/hbase
```

Comment:
nit. I dont think these need to be exported

Reply:
thanks, updated these to shell variables